packages/radix-ui-themes/changelog.md

```diff
@@ -7,6 +7,18 @@
   - Remove comments from the CSS build.
   - Make sure that forced light/dark appearance on the `Theme` component also sets the corresponding browser colors, like the correct input autofill background color.
   - Use `outline` rather than `box-shadow` for most focus styles, which avoids a slight anti-aliasing issue in Chrome on focused elements
+- `TextArea`, `TextField`
+  - Refine padding values for a more balanced look.
+    - Use `text-indent` instead of `padding-left` in the `TextField` so that long values aren't truncated on the left when the cursor is at the end of the input.
+  - Refine how the inner shadows are applied so that they blend with the container background.
+  - Refine and normalise the look and feel of disabled and read-only states.
+    - Improve the background colors in dark mode.
+    - Improve the appearance of placeholders.
+    - Remove visual distinctions between disabled and read-only states.
```

Comment:
The users don't care imo and the differences were superficial and hard to reason about much